website/docs/sql-reference/clauses/sql-limit.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: limit
 title: SQL LIMIT
-description: Adding the LIMIT clause to a query will limit the number of rows returned.
+description: Read this guide to learn about the SQL IN operator in dbt.
```

Comment:
I believe this should say `SQL LIMIT` rather than `SQL IN`

Reply:
https://deploy-preview-3261--docs-getdbt-com.netlify.app/sql-reference/limit

Should now say: Read this guide to learn about the SQL LIMIT clause in dbt.